client/guidestours/index.js

```diff
@@ -183,22 +290,32 @@ export default class GuidesTours extends Component {
 			return null;
 		}
 
-		const { bullseye, dialog } = this.getStepPositions();
-		const stepCoords = posToCss( dialog );
-		const pointerCoords = posToCss( bullseye );
+		let StepComponent = GuidesStep;
+		switch ( this.state.currentStep.type ) {
+			case 'GuidesFirstStep':
```

Comment:
I think we can just leave a `TODO` comment here. I wouldn't want to see this `switch` statement grow any longer.